scripts/build_cloudflare_real_ip_config.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+
+echo "# Cloudflare" > config/00_real_ip_cloudflare_00.conf
+for i in `curl https://www.cloudflare.com/ips-v4`; do
+        echo "set_real_ip_from $i;" >> config/00_real_ip_cloudflare_00.conf
+done
```

Comment:
Is this a recommended way of doing things? Looks as dirty as it could be :)

Reply:
We deploy rather often. So the ips will be bundled into heroku slug. For now, I think it's enough.